data/mods/ssb/random-teams.js

```diff
@@ -634,7 +634,7 @@ class RandomStaffBrosTeams extends RandomTeams {
 				name = this.sampleNoReplace(pool);
 			}
 			let ssbSet = sets[name];
-			if (name === 'Forrce' && this.random()) {
+			if (name === 'Forrce' && this.random(2) === 0) {
```

Comment:
```suggestion
			if (name === 'Forrce' && Math.floor(this.random())) {
```
After testing, `this.random()` generates a random number between 0 and 1 (one of those is inclusive one isnt) like `Math.random`, but it uses the PRNG seed. calling `this.random(2)` would not fix it as theres an even smaller chance of it being 0, and even if you rounded that, it would be 2/3 to not swap then.

Reply:
sounds good